src/components/TypeBase/TypeBase.js

```diff
@@ -168,6 +168,7 @@ export const TypeFoundry = (privateProps) => {
 
   // TODO: figure out if downstream or upstream is the correct nomenclature here
   const PublicTypeComponent = (downstreamProps) => {
+    console.log('downstreamProps props in type foundry: ', downstreamProps)
```

Comment:
Perhaps we can remove this?

Reply:
yup...will be a question New version: 1.1.189 coming soon to reflect